### PR TITLE
Inspec support for ubuntu and rhel

### PR DESCRIPTION
**Pull request by narengan**
Inspec is a testinf framework for compliance and policy requirements. It will
replace sererspec eventually. As of now it will test rhel stigs.

### CODE REVIEW

**Comment by narengan** [comment]
retest

**Comment by narengan** [comment]
retest

**Comment by narengan** [comment]
retest

**Comment by narengan** [comment]
retest

File: roles/inspec/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    mode: 0755
+
+- name: ensure inspec directory exists
+  file:
+    dest: /etc/inspec/controls
+    state: directory
+    owner: root
+    mode: 0755
+
+- name: ensure inspec directory exists
```
Comment (scoop206) [inline roles/inspec/tasks/main.yml]: Can we combine these 3 directory exists actions into one via `with_items` ?

**Comment by narengan** [comment]
retest

**Comment by narengan** [comment]
retest

**Comment by scoop206** [comment]
:+1:

**Pull request merged**